qemu/tests/throttle_operation_test.py

```diff
@@ -0,0 +1,132 @@
+"""IO-Throttling group and other operation relevant testing"""
+import json
+import logging
+
+from provider.block_devices_plug import BlockDevicesPlug
```

Comment:
xchange line5 and line11?
order:
standard lib
virttest(vt)
provider (tp-qemu)